src/wombats/handlers/auth.clj

```diff
@@ -30,6 +30,13 @@
             {:headers {"Authorization" (str "token " access-token)
                        "Accept" "application/json"}}))
 
+(defn- remove-slash
+  "Removes the trailing slash from a url (if it exists)"
+  [url]
+  (if (.endsWith url "/")
```

Comment:
Checkout [ends-with?](https://clojure.github.io/clojure/clojure.string-api.html#clojure.string/ends-with?) for a native function